tests/network/hotplug.go

```diff
@@ -589,3 +540,50 @@ func migrate(vmi *v1.VirtualMachineInstance) {
 	migrationUID := tests.RunMigrationAndExpectCompletion(kubevirt.Client(), migration, tests.MigrationWaitTime)
 	tests.ConfirmVMIPostMigration(kubevirt.Client(), vmi, migrationUID)
 }
+
+func addInterface(vm *v1.VirtualMachine, name, netAttachDefName string) error {
+	newNetwork, newIface := newNetworkInterface(name, netAttachDefName)
+
+	err := tests.RetryIfModified(func() error {
```

Comment:
Please use patching instead of `update`.
The usage of `update` requires retry management, while patching does not.

We have usage of patching in the tests in a structural manner [1].

[1] https://github.com/kubevirt/kubevirt/blob/63233b446fccaf259b7c4b3d5e712ea598aa5a62/pkg/apimachinery/patch/patch.go#L54